conseil-common/src/main/resources/reference.conf

```diff
@@ -25,6 +25,26 @@ platforms {
       }
     }
   }
+
+  bitcoin {
+    mainnet {
+      node {
+        protocol: "http"
+        hostname: "localhost",
+        port: 8332,
+        username: "bitcoin",
+        password: "sP9PV88xtbGwLMAEi2rVlZ7jIFfJbpOmTUCsBBBRN9I="
```

Comment:
Is this somewhat sensitive information to be shared?

Reply:
no, this is a password used in docker-compose

Reply:
where does it come from?

Reply:
I've generated it. Should I change it to "password"?

Reply:
I would go for something simple if it's just for dev purposes

Reply:
I mean, as long as it's nothing referring to a real working infrastructure it's fine. It looked like a real password, so I was worried about security holes, nothing more...